_hooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+
```

Comment:
Fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/3f6000223f6819c18a5133a60e77369a825dff00